app/services/ucas_matches/send_resolved_on_apply_emails.rb

```diff
@@ -0,0 +1,21 @@
+module UCASMatches
+  class SendResolvedOnApplyEmails
+    attr_reader :ucas_match
+
+    def initialize(ucas_match)
+      @ucas_match = ucas_match
+    end
+
+    def call
+      raise 'The application has not been resolved on Apply' if !ucas_match.resolved_on_apply?
+
+      application_choice = ucas_match.application_choices_for_same_course_on_both_services.first
```

Comment:
We are assuming that the candidate only applied for 1 course on both services but theoretically they could have up to 3 application choices for for same course on both services.

Maybe we could add a method called something like `duplicate_application_choices_withdrawn_from_ucas` to the UCASMatch model which would return ApplicationChoices for same course on both services which were withdrawn on UCAS and then send an email for each.
 _Not ideal to spam them with emails but it's an unlikely scenario._

Once we have done that we could change
`if match.ready_to_resolve? && match.ucas_withdrawal?`
to 
`if match.ready_to_resolve? && match.duplicate_application_choices_withdrawn_from_ucas.any?`

Reply:
@AgaDufrat  can we catch up about this tomorrow morning. It seems like there are missing requirements around this functionality that we are trying to compensate for and maybe we should reconsider first reviewing our approach on the initial and reminder emails as well so that we have knowledge of which application we have already emailed the candidate about?

It also sounds like the UR should maybe get involved so we can figure out how to approach sending multiple emails out and if we can instead extend the existing email templates to handle cases of multiple applications.